python/graphstorm/dataloading/dataset.py

```diff
@@ -446,15 +447,28 @@ def prepare_data(self, g):
                 if dist_sum(len(test_idx)) > 0:
                     test_idxs[canonical_etype] = test_idx
             else:
-                print(f"WARNING: {canonical_etype} does not contains " \
-                      "test_mask, skip testing {canonical_etype}")
+                print(f"NOTE: {canonical_etype} does not contains " \
+                      "test_mask, skip testing {canonical_etype}. \n" \
+                      "We will do inference on the whole edge list.")
+                infer_idx = dgl.distributed.edge_split(
+                    th.full((g.num_edges(canonical_etype),), True, dtype=th.bool),
```

Comment:
we should modify DGL to split all edges.

Reply:
You mean DGL provides an API like edge_split_all?

Reply:
maybe something like `edge_split(dgl.ALL, ...)`